src/main/resources/swagger/api-docs.yaml

```diff
@@ -10201,6 +10431,50 @@ components:
         labels:
           type: string
           description: The custom labels as json
+    BillingProjectResponse:
+      required:
+        - projectName
+        - invalidBillingAccount
+        - roles
+      type: object
+      properties:
+        projectName:
+          type: string
+          description: the name of the project
+        billingAccount:
+          type: string
+          description: the billing account to use in google projects
+        servicePerimeter:
+          type: string
+          description: the name of the service permeters for google project
+        invalidBillingAccount:
+          type: boolean
+          description: whether or not the billing account is usable by Terra
+        roles:
+          type: array
+          items:
+            type: string
+          description: the roles the caller has on the project
+        workspacesWithCorrectBillingAccount:
```

Comment:
Adding these two fields and `WorkspaceBillingAccount` are the only two manual changes I made, everything else was from the merge command.